src/ducks/transactions/TransactionsPage.jsx

```diff
@@ -79,26 +85,52 @@ const Separator = () => (
   <hr style={{ fontSize: 0, border: 0, height: '0.5rem' }} />
 )
 
+const getMonth = date => date.slice(0, 7)
```

Comment:
Why `slice(0, 7)` ?

Reply:
dates are strings in the transactions, ex: 2014-08-25T08:00, so a slice until the 7th character will only take the month part 2014-08.